instrumentation/aws-sdk/aws-sdk-1.11/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v1_11/AwsSdkAttributesExtractor.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.awssdk.v1_11;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+
+import com.amazonaws.Request;
+import com.amazonaws.Response;
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpClientAttributesExtractor;
+import java.util.List;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public class AwsSdkAttributesExtractor
```

Comment:
Most of these classes should probably be package private - we shouldn't expose them as part of the library's public API.